README.md

```diff
@@ -304,3 +298,27 @@ ZipPart.all.annotate(caller).where.not(status: 'ok').count
 ZipPart.annotate(caller).where.not(status: 'ok').count
 ZipPart.where.not(status: 'ok').count.annotate(caller)
 ```
+
+## Resetting the preservation system
```

Comment:
generalized from https://github.com/sul-dlss/argo/issues/4052#issuecomment-1690550978

omits the stuff about disentangling conflated QA and stage storage roots, since hopefully we won't make that mistake again 🙂